app/communication/models/banner.py

```diff
@@ -0,0 +1,57 @@
+import uuid
+
+from django.db import models
+from django.db.models import Q
+from django.utils import timezone
+
+from app.common.enums import AdminGroup
+from app.common.permissions import BasePermissionModel
+from app.util.models import BaseModel, OptionalImage
+from app.util.utils import now
+
+
+def tomorrow():
+    return timezone.now() + timezone.timedelta(days=1)
+
+
+class Banner(BaseModel, OptionalImage, BasePermissionModel):
+    write_access = AdminGroup.admin()
+
+    id = models.UUIDField(
+        auto_created=True,
+        primary_key=True,
+        default=uuid.uuid4,
+        serialize=False,
+    )
+    title = models.CharField(max_length=100)
+    description = models.TextField()
+    visible_from = models.DateTimeField(default=timezone.now)
+    visible_until = models.DateTimeField(default=tomorrow)
+    url = models.URLField(max_length=600, blank=True, null=True)
+
+    def __str__(self):
+        return f"{self.title} {'- is visible' if self.is_visible else ''} - {self.description}"
+
+    def save(self, *args, **kwargs):
+        if self.is_visible and self.is_uniquely_visible:
+            raise ValueError("Det finnes allerede et banner som er synlig")
+        if self.visible_from > self.visible_until:
+            raise ValueError("Datoer er satt feil")
+        super().save(*args, **kwargs)
+
+    class Meta:
+        ordering = ("-updated_at",)
+
+    @property
+    def is_visible(self):
+        return self.visible_from <= now() and self.visible_until >= now()
+
+    @property
+    def is_uniquely_visible(self):
+        return (
+            Banner.objects.filter(
+                Q(visible_from__lte=now()) & Q(visible_until__gte=now())
+            )
```

Comment:
Dette er det samme som å skrive 
```suggestion
            Banner.objects.filter(
                visible_from__lte=now(), visible_until__gte=now()
            )
```
da kommaseparerte argumenter i filteret vil AND-es i SQL-setningen😉